docs/docs/getting-started/concepts.md

```diff
@@ -0,0 +1,641 @@
+| --- | --- |
+| [definitions](concepts#definitions) | `code location` must contain at least one `definitions` |
+
+### Config
+
+```mermaid
+%%{
+  init: {
+    'theme': 'base',
+    'themeVariables': {
+      'primaryColor': '#4F43DD',
+      'primaryTextColor': '#fff',
+      'primaryBorderColor': '#231F1B',
+      'lineColor': '#DEDDFF',
+      'secondaryColor': '#BDBAB7',
+      'tertiaryColor': '#BDBAB7'
+    }
+  }
+}%%
+  graph LR
+    Config(Config)
+
+    style Job fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Asset fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Schedule fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Sensor fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+
+    Config -.-> Asset
+    Config -.-> Job
+    Config -.-> Schedule
+    Config -.-> Sensor
+```
+
+A <PyObject section="config" module="dagster" object="RunConfig" /> is a set schema applied to a Dagster object that is input at the time of execution. This allows for parameterization and the reuse of pipelines to serve multiple purposes.
+
+| Concept | Relationship |
+| --- | --- |
+| [asset](concepts#asset) | `config` may be used by an `asset` |
+| [job](concepts#job) | `config` may be used by a `job` |
+| [schedule](concepts#schedule) | `config` may be used by a `schedule` |
+| [sensor](concepts#sensor) | `config` may be used by a `sensor` |
+
+### Definitions
+
+```mermaid
+%%{
+  init: {
+    'theme': 'base',
+    'themeVariables': {
+      'primaryColor': '#4F43DD',
+      'primaryTextColor': '#fff',
+      'primaryBorderColor': '#231F1B',
+      'lineColor': '#DEDDFF',
+      'secondaryColor': '#BDBAB7',
+      'tertiaryColor': '#BDBAB7'
+    }
+  }
+}%%
+  graph LR
+    style Job fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Schedule fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Asset fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Sensor fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style IOManager fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Resource fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style AssetCheck fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style CodeLocation fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+
+
+    Asset -.-> Definitions
+    AssetCheck -.-> Definitions
+    IOManager -.-> Definitions
+    Job -.-> Definitions
+    Resource -.-> Definitions
+    Schedule -.-> Definitions
+    Sensor -.-> Definitions
+
+    Definitions(Definitions)
+
+    Definitions ==> CodeLocation
+```
+
+A <PyObject section="definitions" module="dagster" object="Definitions" /> is a top-level construct containing all the objects of a Dagster project, such as <PyObject section="assets" module="dagster" object="asset" decorator />, <PyObject section="jobs" module="dagster" object="job" decorator /> and <PyObject section="schedules-sensors" module="dagster" object="ScheduleDefinition" />. Only objects included in the definitions will be deployed and visible within the Dagster UI.
```

Comment:
I would tweak this slightly for readability: "Definitions is a top-level construct that contains references to all the objects of a Dagster project...."

Reply:
You can use `pluralize` in `PyObject` to make the objects plural, e.g. `<PyObject section="assets" module="dagster" object="asset" pluralize />` (same goes for the other PyObjects on this page)